src/liveShareSupport.ts

```diff
@@ -0,0 +1,64 @@
+import { Disposable } from 'vscode';
+import * as vsls from 'vsls';
+
+let liveShare: vsls.LiveShare = null;
+let liveShareListener: Disposable = null
+let connectedPort: number = null;
+let jackedIn = false
```

Comment:
I think this stateful variable can be removed. I only see it being set, but never read, unless I'm missing something.

Reply:
I added a comment in the code to clarify:

```typescript
// Keeps hold of the LiveShare API instance, so that it is requested only once.
let liveShare: vsls.LiveShare = null;

// Keeps hold of the LiveShare listener, to prevent it from being disposed immediately.
let liveShareListener: Disposable = null;
```